1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,11 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents = group.every(any);{
+return students.includes(any);
+}
+console.log(groupIsOnlyStudents);
+
```

Comment:
This you need to define 'any', and maybe rethink what is going to go inside .every().
We also need to create a function if we want to return something.
.every() takes an array and checks if every element passes a certain test - this test can be a function - I have a feeling this is what you were trying to do.

Reply:
Thank you 🙏 I see